modelling/src/main/java/org/axonframework/modelling/saga/AbstractSagaManager.java

```diff
@@ -161,25 +164,27 @@ protected boolean matchesSegment(Segment segment, String sagaId) {
      * Returns the Saga Initialization Policy for a Saga of the given {@code sagaType} and {@code event}. This policy
      * provides the conditions to create new Saga instance, as well as the initial association of that saga.
      *
-     * @param event The Event that is being dispatched to Saga instances
+     * @param event   The Event that is being dispatched to Saga instances
+     * @param context
      * @return the initialization policy for the Saga
      */
-    protected abstract SagaInitializationPolicy getSagaCreationPolicy(EventMessage<?> event);
+    protected abstract SagaInitializationPolicy getSagaCreationPolicy(EventMessage<?> event, ProcessingContext context);
 
     /**
      * Extracts the AssociationValues from the given {@code event} as relevant for a Saga of given {@code sagaType}. A
      * single event may be associated with multiple values.
      *
-     * @param event The event containing the association information
+     * @param event   The event containing the association information
+     * @param context
```

Comment:
As we will surely move these classes to the legacy/migration package, this new parameter needs some JavaDoc.

Reply:
Consider it added

Reply:
Javadoc has been added to this and relevant methods. Resolving.